cicecore/cicedyn/dynamics/ice_transport_remap.F90

```diff
@@ -2011,6 +2012,13 @@ subroutine locate_triangles (nx_block,     ny_block,   &
                icellsd = icellsd + 1
                indxid(icellsd) = i
                indxjd(icellsd) = j
+            else ! temp comment: dont need if (l_fixed_area) because edgearea=0 when F
```

Comment:
I mean here and the other places with temp comment. 

We should keep the comment:
! 1 grid-cell wide channel: dpx,y = 0, edgearea /= 0